src/Interface/Modules/Base/HasParserHelpDialog.cc

```diff
@@ -33,8 +33,7 @@ using namespace SCIRun::Gui;
 
 void ModuleDialogWithParserHelp::popUpParserHelp()
 {
-  if (!help_)
-    help_ = new ParserHelpDialog(this);
+  help_ = new ParserHelpDialog(this);
```

Comment:
This line (creating a new instance every time) leads to multiple dialogs being created each time the user clicks the show button. If you add back the null check, that will take care of only creating and showing one instance per module.